cfgov/jinja2/v1/_includes/organisms/simple-chart.html

```diff
@@ -23,11 +23,11 @@ <h5>Figure {{ value.figure }}</h5>
     {% endif %}
 
     {% if value.title %}
-    <h2>{{ value.title }}</h2>
+    <h2 class="simple-chart-title">{{ value.title }}</h2>
```

Comment:
Should classes be BEMified in this organism? Looks like this should maybe be `o-simple-chart_title`, for example.